en/6-use-open-standards-solutions.md

```diff
@@ -121,6 +121,8 @@ Using open standards and common government platforms helps you to:
 ### Similar resources
 
 - [Open Standards - Open First Whitepaper (GC)](https://github.com/canada-ca/Open_First_Whitepaper/blob/master/en/2_Open_Standards.md)
+- [3. Be open and use open source (Technology Code of Practice (UK))](https://www.gov.uk/guidance/be-open-and-use-open-source)

```

Comment:
Not sure if it's intentional, but the numbering here looks odd

Reply:
It seems that the UK has numbers in the page title for their Digital Service Standards but not their Technology Code of Practice, even though both are numbered in the table of contents. Providing the numbering is consistent though with how we've been referring to numbered standards/codes/plays in source material.